### PR TITLE
[JIRA GEOS-9031][Backup & Restore] Add the possibility of filtering against Store and Layer

### PR DESCRIPTION
**Pull request by afabiani**

### CODE REVIEW

File: src/security/ldap/pom.xml [hunk 1]
```diff
@@ -40,6 +40,16 @@
       <groupId>org.springframework.ldap</groupId>
       <artifactId>spring-ldap-test</artifactId>
       <scope>test</scope>
+      <exclusions>
```
Comment (mbarto) [inline src/security/ldap/pom.xml]: Why do we need this?

Comment (afabiani) [reply src/security/ldap/pom.xml]: The "ldap-tests" module includes a very old version of Spring-Batch which is incompatible with the version used by the Backup-&-Restore module. You might end up with a method clashing and odd errors like "method not found" on Classes.  In some circumstances, it also prevents GeoServer to start.

Comment (mbarto) [reply src/security/ldap/pom.xml]: Good, merging then. I see you already backported to 2.13.x, can you also backport for 2.14.x?

Comment (afabiani) [reply src/security/ldap/pom.xml]: Sure

**Pull request merged**